packages/amplify-category-analytics/tests/commands/console.test.js

```diff
@@ -0,0 +1,11 @@
+const console = require('../../commands/analytics/console');
+
+jest.mock('../../index', () => ({ console: () => {} }));
+
+describe('analytics console: ', () => {
+  it('enable run method should exist', () => {
+    expect(console.run).toBeDefined();
+    expect(console.name).toBeDefined();
```

Comment:
You might want to add a assertion to see of the `console` method from the `index.js` is called. And you mught as well get rid of `console.run` toBeDefined